### DISCUSSION TITLE
How to train Faster R-CNN on my own custom dataset and changing the RPN loss functions?

### DISCUSSION THREAD

**Discussion by meshghi** [facebookresearch/detectron2#3407] [Q&A]
Hello. I want to:

1- train Faster RCNN on my own custom dataset.
2- use the pre-trained Faster RCNN on the VOC2007 as the initial weights to train it then on my own custom dataset.
3- modify the RPN network with different loss functions that I have come up with.

I will greatly appreciate it if you could help me/let me know how to achieve the above goals.
